spec/features/reviews/02_user_votes_review_spec.rb

```diff
@@ -24,13 +24,13 @@
     user_sign_in(user)
     visit lifehack_path(review.lifehack)
     within('.review') do
-      expect(page).to have_content("Score: 0")
+      expect(page).to have_content("0")
     end
     click_button '+1'
 
     expect(page).to have_content('Vote cast!')
     within('.review') do
```

Comment:
could be more specific with the css selector to make sure it's testing for the right place

maybe within('.review td.rev_score')